src/components/DpConfirmDialog/DpConfirmDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <dp-modal
+    ref="confirmDialog"
+    content-classes="w-2/3 xl:w-1/3"
+    :data-cy="`confirmDialog:${dataCy}`"
+    aria-describedby="dialogDescription"
+    @modal:toggled="onModalToggled">
+    <p
+      id="dialogDescription"
+      class="break-words mb-3">
+      {{ message }}
+    </p>
+    <dp-button-row
+      primary
+      :primary-text="confirmButtonText"
+      secondary
+      @primary-action="handleDialogResponse(true)"
+      @secondary-action="handleDialogResponse(false)" />
+  </dp-modal>
+</template>
+
+<script lang="ts">
+import { de } from '~/components/shared/translations'
+import DpButtonRow from '~/components/DpButtonRow'
+import DpModal from '~/components/DpModal'
+
+export default {
+  name: 'DpDialog',
+
+  components: {
+    DpButtonRow,
+    DpModal
+  },
+
+  props: {
+    dataCy: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    message: {
+      type: String,
+      required: false,
+      default: de.dialog.default
+    }
+  },
+
+  data () {
+    return {
+      confirmButtonText: de.operations.confirm,
+      resolvePromise: null as ((result: boolean) => void) | null
+    }
+  },
+  methods: {
+    handleDialogResponse (isConfirmed: boolean): void {
```

Comment:
Nitpick: i'd call this `handleConfirm` (just a personal preference)

Reply:
yes, this option is better --> https://github.com/demos-europe/demosplan-ui/pull/1228/commits/723c95af6ed364f74004b5456fa8d177bab7ad4d